model/coverage.py

```diff
@@ -664,9 +663,15 @@ def bulk_add(
         update = (
             WorkCoverageRecord.__table__.update()
             .where(
-                and_(
-                    WorkCoverageRecord.work_id.in_(work_ids),
-                    WorkCoverageRecord.operation == operation,
+                WorkCoverageRecord.id.in_(
```

Comment:
It might be helpful to add a comment about what you are aiming to do with this more “involved” update.

Reply:
I added a comment.